File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdRetailIdPackagesInfoPackagesItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class RetailIdRetailIdPackagesInfoPackagesItem(
    @JsonProperty("EstimatedBalance")
    val estimatedBalance: Int? = null,
    @JsonProperty("HasQrs")
    val hasQrs: Boolean? = null,
    @JsonProperty("IssuanceId")
    val issuanceId: String? = null,
    @JsonProperty("Issuances")
    val issuances: List<RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem>? = null,
    @JsonProperty("QrCount")
    val qrCount: Int? = null,
    @JsonProperty("RequiresVerification")
    val requiresVerification: Boolean? = null,
    @JsonProperty("SiblingCount")
    val siblingCount: Int? = null,
    @JsonProperty("Tag")
    val tag: String? = null
)

File: sdks/thunkmetrc-go/wrapper/models/retail_id_retail_id_packages_info_packages_item.go
package models

type RetailIdRetailIdPackagesInfoPackagesItem struct {
    EstimatedBalance int64 `json:"EstimatedBalance,omitempty"`
    HasQrs bool `json:"HasQrs,omitempty"`
    IssuanceId string `json:"IssuanceId,omitempty"`
    Issuances []RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem `json:"Issuances,omitempty"`
    QrCount int64 `json:"QrCount,omitempty"`
    RequiresVerification bool `json:"RequiresVerification,omitempty"`
    SiblingCount int64 `json:"SiblingCount,omitempty"`
    Tag string `json:"Tag,omitempty"`
}
